From 32435e37d430df3a0471794ef90e0a1e385f7503 Mon Sep 17 00:00:00 2001
From: Joseph Myers <joseph@codesourcery.com>
Date: Wed, 10 Aug 2016 07:43:01 -0700
Subject: [PATCH 06/16] Add release note for increased Linux kernel version
 requirement.

	* release-notes-csl.xml (Linux kernel 3.2 or later required): New
	note.
---
 ChangeLog.csl         |  5 +++++
 release-notes-csl.xml | 12 ++++++++++++
 2 files changed, 17 insertions(+)

diff --git a/ChangeLog.csl b/ChangeLog.csl
index 9997a70..eff3e8b 100644
--- a/ChangeLog.csl
+++ b/ChangeLog.csl
@@ -1,3 +1,8 @@
+2016-08-10  Joseph Myers  <joseph@codesourcery.com>
+
+	* release-notes-csl.xml (Linux kernel 3.2 or later required): New
+	note.
+
 2016-08-09  Joseph Myers  <joseph@codesourcery.com>
 
 	Merge from SCB glibc 2.23:
diff --git a/release-notes-csl.xml b/release-notes-csl.xml
index e8bc0cd..b33ed3f 100644
--- a/release-notes-csl.xml
+++ b/release-notes-csl.xml
@@ -5,6 +5,18 @@
      relevant; it may be removed if it is relevant to all targets.
  -->
 
+      <!-- UUID: 2cb5e43f-f852-4c25-8e0f-889ad8894059 -->
+      <!-- TARGET: arm-none-linux-gnueabi powerpc-linux-gnu powerpc-lsi-linux-gnu -->
+      <!-- SORT: glibc -->
+      <!-- ISSUE: -->
+      <formalpara>
+        <title>Linux kernel 3.2 or later required</title>
+        <para>
+	  The GNU C Library included in &csl_prod; for
+	  &csl_target_name; now requires Linux 3.2 or later.
+        </para>
+      </formalpara>
+
       <!-- UUID: 30e1ed92-d1f8-40fb-b14c-9f90ff68eab4 -->
       <!-- TARGET: arm-none-linux-gnueabi -->
       <!-- SORT: glibc -->
-- 
2.8.1

